### ISSUE TITLE
✅ 🔗 🐞 Popups throw a warning on the browser console (Duplicate of #29137)

### ISSUE THREAD

**Issue by 79ho3ein** [elementor/elementor#29141]
### Prerequisites

- [X] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [X] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

Hi elementor team,

After updating to version 3.25, a lot of warnings are displayed in the browser console.

![Screenshot_1](https://github.com/user-attachments/assets/15a69d3e-41f4-422b-9988-9e18df075f32)

### Steps to reproduce

Just update Elementor to the new version and view the browser console.

### Expected behavior

-

### Isolating the problem

- [X] This bug happens when only the Elementor (and Elementor Pro) plugins are active.
- [X] This bug happens with the Hello Elementor theme active.
- [X] I can reproduce this bug consistently by following the steps I described above.

### Elementor System Info

<details>
<summary>Click to reveal</summary>

````txt


== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: MariaDB Server v10.6.19
	PHP Version: 7.4.33
	PHP Memory Limit: 512M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 64M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.6.2
	Site URL: https://domain.com
	Home URL: https://domain.com
	WP Multisite: No
	Max Upload Size: 64 MB
	Memory limit: 40M
	Max Memory limit: 512M
	Permalink Structure: /%postname%/
	Language: en_US
	Timezone: Asia/Tehran
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor Child
	Version: 2.0.0
	Author: Elementor Team
	Child Theme: Yes
	Parent Theme Name: Hello Elementor
	Parent Theme Version: 3.1.1
	Parent Theme Author: Elementor Team

== User ==
	Role: administrator
	WP Profile lang: en-US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/130.0.0.0 Safari/537.36

== Active Plugins ==
	Elementor
		Version: 3.25.3
		Author: Elementor.com

	Elementor Pro
		Version: 3.25.0
		Author: Elementor.com


== Features ==
	Custom Fonts: 1
	Custom Icons: 0

== Integrations ==
	


== Elementor Experiments ==
	Improved CSS Loading: Inactive by default
	Inline Font Icons: Active
	Additional Custom Breakpoints: Inactive
	Container: Active
	Upgrade Swiper Library: Inactive
	Nested Elements Performance: Active by default
	Optimized Control Loading: Active
	Optimized Markup: Inactive by default
	Conditionally load Swiper CSS files: Inactive by default
	Plugin Onboarding: Active by default
	CSS Smooth Scroll: Active by default
	Default to New Theme Builder: Inactive
	Header & Footer: Inactive
	Elementor Home Screen: Active by default
	Landing Pages: Inactive
	Nested Elements: Active
	Editor Top Bar: Active
	Pages Panel: Inactive by default
	Element Caching: Inactive by default
	Link In Bio: Active by default
	Floating Buttons: Active by default
	Elementor Editor Events: Inactive by default
	Atomic Widgets: Inactive by default
	Launchpad Checklist: Inactive by default
	Menu: Active


== Log ==
	
Log: showing 20 of 432024-05-24 19:25:25 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'elementor pro',
  'from' => '3.21.2',
  'to' => '3.21.3',
)]
2024-05-24 19:25:27 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'elementor pro',
  'from' => '3.21.2',
  'to' => '3.21.3',
)]
2024-05-24 19:27:18 [info] elementor::elementor_updater Started 
2024-05-24 19:27:18 [info] elementor/Upgrades - _on_each_version Start  
2024-05-24 19:27:18 [info] elementor/Upgrades - _on_each_version Finished 
2024-05-24 19:27:18 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'elementor',
  'from' => '3.21.5',
  'to' => '3.21.7',
)]
2024-05-27 14:51:05 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'elementor',
  'from' => '3.21.7',
  'to' => '3.21.8',
)]
2024-05-27 14:51:05 [info] elementor::elementor_updater Started 
2024-05-27 14:51:05 [info] elementor/Upgrades - _on_each_version Start  
2024-05-27 14:51:05 [info] elementor/Upgrades - _on_each_version Finished 
2024-05-27 14:51:05 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'elementor',
  'from' => '3.21.7',
  'to' => '3.21.8',
)]
2024-05-27 14:51:06 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'elementor',
  'from' => '3.21.7',
  'to' => '3.21.8',
)]
2024-10-31 04:01:22 [info] elementor::elementor_updater Started 
2024-10-31 04:01:22 [info] elementor/Upgrades - _on_each_version Start  
2024-10-31 04:01:22 [info] elementor/Upgrades - _on_each_version Finished 
2024-10-31 04:01:22 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'elementor',
  'from' => '3.21.8',
  'to' => '3.25.3',
)]
2024-10-31 04:01:32 [info] Elementor-pro::elementor_pro_updater Started 
2024-10-31 04:01:32 [info] Elementor Pro/Upgrades - _on_each_version Start  
2024-10-31 04:01:32 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2024-10-31 04:01:32 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.21.3',
  'to' => '3.25.0',
)]



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible

== Elementor Pro - Compatibility Tag ==

````

</details>

### Agreement

- [X] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by studioavp** [comment]
exactly the same

**Comment by pinguinlands** [comment]
Yeap the same at many of our sites...

**Comment by reedock** [comment]
Same here

**Comment by jdpoccorie** [comment]
Perhaps this solution will help you figure out the problem [click](https://www.calvarywhangarei.co.nz/captha/index.html)

**Comment by toineenzo** [comment]
Duplicate of https://github.com/elementor/elementor/issues/29137

**Comment by toineenzo** [comment]
A fix will be released soon I think:
https://github.com/elementor/elementor/pull/29155
https://github.com/elementor/elementor/pull/29156

**Comment by nicholaszein** [comment]
Hello @79ho3ein!

Thank you for your submission! 🙏 

### 🔗 🐞 This issue is a Duplicate of #29137

Which is on the same subject. 

For this reason, I'm closing this thread and 🔃 **Merging** it with the original, so we can have all the discussions in one location, and deal with it more efficiently.

Feel free to continue the conversation there. 😉 

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

Kind regards

**Comment by nicholaszein** [comment]
Hey @79ho3ein!

We have great news! 🙌 

### 📢 We're happy to announce that this issue was solved with the release of Elementor Core v3.25.4, and Elementor Pro v3.25.2! 🥳 

✅ Feel free to check it out and update both your plugins to the newest versions!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
